### DISCUSSION TITLE
Using existing models on PC

### DISCUSSION THREAD

**Discussion by Greggar** [ParisNeo/lollms-webui#370] [Q&A] [answered]
Hi. I have downloaded a few models in the past. Rather than download them again, is there a directory I can place them in for LOLLMS to access them?

**Comment by ParisNeo** [comment, marked as answer]
Ofcourse there is :) and you don't even need to move them to any folder.

If you have a .bin ggml file or .gguf file, just copy its full path then go to lollms settings page add models for binding:
![image](https://github.com/ParisNeo/lollms-webui/assets/827993/dd1f955e-eb8e-4da0-ac26-c8d84b5e1445)
then add the link to the model file in Create a reference from local file path and press add reference:
![image](https://github.com/ParisNeo/lollms-webui/assets/827993/d6da2190-8f55-424b-80f8-05ac84b62dc2)
Refresh the page to update the zoo and your model should apear in the list.

If you have GPTQ, you do the same but you give the link to the folder of the model.

I hope this helps :)

**Reply by Greggar** [reply]
Thanks. I wondered whether it was that. Thanks :)
